docker-example/README.md

```diff
@@ -1,37 +1,64 @@
-# Reflex Container Image Build
+# Reflex Docker Container
 
 This example describes how to create and use a container image for Reflex with your own code.
 
 ## Update Requirements
 
-The `requirements.txt` includes the reflex package which is need to install Reflex framework. If you use additional packages in your project you have add this in the `requirements.txt` first. Copy the `Dockerfile` and the `requirements.txt` file in your project folder.
+The `requirements.txt` includes the reflex package which is needed to install
+Reflex framework. If you use additional packages in your project you have to add
+this in the `requirements.txt` first. Copy the `Dockerfile`, `.dockerignore` and
+the `requirements.txt` file in your project folder.
 
-## Customize Reflex Config
+## Build Reflex Container Image
+
+To build your container image run the following command:
+
+```bash
+docker build -t reflex-app:latest . --build-arg API_URL=http://app.example.com:8000
+```
 
-The `rxconfig.py` includes the configuration of your Reflex service. Edit the file like the following configuration. If you want to use a custom database you can set the endpoint in this file. Ensure that `api_url` points to the publicly accessible hostname where the container will be running.
+Ensure that `API_URL` is set to the publicly accessible hostname or IP where the app
+will be hosted.
 
-```python
-import reflex as rx
+## Start Container Service
 
-config = rx.Config(
-    app_name="app",
-    api_url="http://app.example.com:8000",
-    db_url="sqlite:///reflex.db",
-)
+Finally, you can start your Reflex container service as follows:
```

Comment:
Maybe we should add a note that it will take some time for the services to show up at ports 3000 8000. Or can we make the docker container forward the logs? Since there's no feedback it's hard to know when it has started

Reply:
Or we could take the `-d` off the command here and then the logs would just stream to the console. More experienced docker users would know that `-d` could be added (along with some health checks) to background the container and wait for it to come up.

I got some feedback from another user on discord that we probably shouldn't be recommending `-d` in the example command.

Reply:
Ah, yeah good call taking off the `-d` they can add that back if they want